Set the install_name.
Don't search in directories we didn't specify.
--- configure.orig	2018-10-20 05:03:18.000000000 -0500
+++ configure	2018-11-15 21:56:02.000000000 -0600
@@ -6896,7 +6896,7 @@
   darwin*)
     SHRLIB="libwcs.$LIBVER.dylib"
     SONAME="libwcs.$SHVER.dylib"
-    SHRLD="$SHRLD -dynamiclib -single_module"
+    SHRLD="$SHRLD -dynamiclib -install_name $prefix/lib/$SONAME -single_module"
     SHRLD="$SHRLD -compatibility_version $SHVER -current_version $LIBVER"
     SHRLN="libwcs.dylib"
 
@@ -7427,14 +7427,6 @@
   if test "x$with_cfitsioinc" != x ; then
     CFITSIO_INCDIRS="$with_cfitsioinc"
   fi
-
-  CFITSIO_INCDIRS="$CFITSIO_INCDIRS   \
-           /usr/local/cfitsio/include \
-           /local/cfitsio/include"
-
-  LIBDIRS="$LIBDIRS           \
-           /usr/local/cfitsio/lib \
-           /local/cfitsio/lib"
 fi
 
 
@@ -7466,26 +7458,11 @@
   if test "x$with_pgplotinc" != x ; then
     PGPLOT_INCDIRS="$with_pgplotinc"
   fi
-
-  PGPLOT_INCDIRS="$PGPLOT_INCDIRS    \
-           /usr/local/pgplot/include \
-           /local/pgplot/include"
-
-  LIBDIRS="$LIBDIRS           \
-           /usr/local/pgplot/lib  \
-           /local/pgplot/lib"
 fi
 
 
 if test "x$with_cfitsio" != xno -o \
         "x$with_pgplot"  != xno ; then
-  LIBDIRS="$LIBDIRS           \
-           /usr/local/lib     \
-           /local/lib         \
-           /opt/local/lib     \
-           /opt/SUNWspro/lib  \
-           /sw/lib"
-
   for LIBDIR in $LIBDIRS ; do
     as_ac_File=`$as_echo "ac_cv_file_$LIBDIR" | $as_tr_sh`
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for $LIBDIR" >&5
@@ -7513,12 +7490,7 @@
   done
 
   # Generic include directories.
-  INCDIRS="/usr/local/include \
-           /local/include     \
-           /opt/local/include \
-           /sw/include        \
-           /local             \
-           /usr/include"
+  INCDIRS=
 
 
   # CFITSIO.
